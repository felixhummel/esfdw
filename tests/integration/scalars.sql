SELECT
  _id,
  some_id,
  my_string,
  foo
FROM hello

