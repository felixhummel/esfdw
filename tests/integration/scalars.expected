2|2|Hello World 2!|bar
1|1|Hello World 1!|bar
